### DISCUSSION TITLE
Microsoft.Extensions.DependencyInjection.Abstractions version

### DISCUSSION THREAD

**Discussion by uxohotteixo** [alirezanet/Gridify#143] [Q&A] [answered]
Hi! Is there any way to raise the version of Microsoft.Extensions.DependencyInjection.Abstractions packages for netstandard2.1 to version 3.1.32? Right now I can't use Gridify in a project that already has Microsoft.Extensions.DependencyInjection.Abstraction version higher than 3.0.0 installed :(

For example in the netstandard2.1 project I need EF Core 3.1.11 and Gridify together. But EF Core 3.1.11 has a dependency on Microsoft.Extensions.DependencyInjection.Abstractions version 3.1.11. So I don't know what to do because 3.1.11 is higher than the Gridify limitation for netstandard2.1 which allows Microsoft.Extensions.DependencyInjection.Abstraction to version 3.0.0.

**Comment by alirezanet** [comment]
Hi @uxohotteixo,
I'll release a new version to fix this problem tonight.

**Reply by uxohotteixo** [reply]
@alirezanet 
Thank you very much! Would it be possible for netstandard2.1 Gridify to make this version range of Microsoft.Extensions.DependencyInjection.Abstractions >= 2.1.0 or >=3.0.0?

So with no version restriction on top as it is now.
![image](https://github.com/alirezanet/Gridify/assets/22033601/9077b3b7-11ca-427a-bb31-bc0a9b5e20bf)

**Reply by alirezanet** [reply, marked as answer]
Hi @uxohotteixo,
Can you try `v2.14.1` please and let me know if it fixed your problem?

**Reply by uxohotteixo** [reply]
Yeap thanks!!! New version fixed my problem!
